org.gnome.Crosswords.PuzzleSets.xword-dl.json

```diff
@@ -3,7 +3,7 @@
     "runtime": "org.gnome.Crosswords",
     "runtime-version": "stable",
     "branch": "1",
-    "sdk" : "org.gnome.Sdk//46",
+    "sdk" : "org.gnome.Sdk//47",
     "build-extension": true,
     "separate-locales": false,
     "appstream-compose": false,
```

Comment:
you have to drop this too, otherwise nothing is composed. The idea is that with newer runtimes it works by default for extensions without needing a custom command (same as how it worked for apps)

Reply:
same in any extension you maintain.